server/db.js

```diff
@@ -3,7 +3,8 @@ import * as dotenv from "dotenv";
 
 dotenv.config();
 
-const dbUrl = process.env.MONGODB_URI || "mongodb://localhost:27017/cyf";
+const dbUrl = process.env.MONGODB_URI;
```

Comment:
Is the localhost commented out for testing purposes?

Reply:
Testing purposes. Forgot to rewind :)